backend/package.json

```diff
@@ -8,19 +8,21 @@
     "lint:fix": "eslint -c .eslintrc.js \"src/**/*.{js,ts,tsx}\" --quiet --fix",
     "build": "tsc",
     "build:container": "tsc",
-    "start": "node dist/src/index.js",
+    "start": "npx prisma migrate deploy && node dist/src/index.js",
     "dev": "NODE_ENV=dev tsx src/index.ts",
     "dev:watch": "NODE_ENV=dev tsx watch src/index.ts",
     "test": "jest --coverage --verbose",
     "ts-node": "ts-node"
   },
   "dependencies": {
+    "@prisma/client": "^4.15.0",
     "cors": "^2.8.5",
     "envsafe": "^2.0.3",
     "express": "^4.18.2",
     "ioredis": "^5.3.2",
     "jsonwebtoken": "^9.0.0",
     "pg": "^8.10.0",
+    "prisma": "^4.15.0",
```

Comment:
I think this is a dev dependency

Reply:
donezo, i think we spotted this issue before but looks like i forgot to move it down 😭